### PR TITLE
Changed snap link

### PR DESCRIPTION
**Pull request by ito32bit**
change from Building status to Snap Store

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,6 +69,7 @@ It can be installed using yay as shown below.
 ```
 yay -S lxqt-organizer
 ```
+
```
Comment (shlyakpavel) [inline README.md:72]: Why?

File: README.md [hunk 2]
```diff
@@ -52,7 +52,7 @@ qtmultimedia5-dev
 
 Binary packages are available as snap packages and in the Arch User Repository (AUR) 
 
-* [Snap by ito32bit](https://build.snapcraft.io/user/ito32bit/lxqt-organizer-snap-packaging) 
+* [Snap by ito32bit](https://snapcraft.io/lxqt-organizer-snap)
```
Comment (shlyakpavel) [inline README.md:55]: Looks sane

File: README.md (hunk 1 above)
Comment (shlyakpavel) [reply README.md:72]: I don't see any reason for a newline. In case there is, @ito32bit please explain

**Review by shlyakpavel — APPROVED** [review]
LGTM

**Comment by tsujan** [comment]
Will be archived.

**Pull request closed without merging**